portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -180,6 +180,39 @@ definition {
 		}
 	}
 
+	@description = "LRQA-55802, Validation of images for Sign In and Sign Out "
+	@priority = "5"
+	test ValidateSignInSignOut {
+		property portal.acceptance = "false";
+		property test.name.skip.portal.instance = "UIInfrastructureUsecase#RefreshWebContent";
+
+		SignOut.signOut();
+
+		Refresh();
+
+		takeScreenshot();
+
+		SikuliAssertElementPresent(locator1 = "Navigation#SCREENSHOTS_SIGN_IN");
+
+		Navigator.openSpecificURL(url = "http://localhost:8080/c/portal/login");
+
+		SikuliAssertElementPresent(locator1 = "Navigation#SCREENSHOTS_LOG_IN_MODAL");
+
+		SignIn.signInSpecificURL(url = "http://localhost:8080/c/portal/login");
+
+		Refresh();
+
+		takeScreenshot();
+
+		AssertElementPresent(locator1 = "UserBar#USER_AVATAR_ICON_IE");
```

Comment:
So the Internet explorer icon works here? Did you try the regular USER_AVATAR_ICON? I have a feeling that either one would work, but for proper syntax let's use USER_AVATAR_ICON instead since we're not specifically testing IE11. We may want to create a separate ticket to investigate if IE icon path is safe enough to replace the regular path for all browsers in all other tests.